<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>保存位置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathEdit">
        <property name="placeholderText">
         <string>截图保存路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectDirButton">
        <property name="text">
         <string>选择位置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>快速截图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fastCaptureEdit">
        <property name="text">
         <string>alt+z</string>
        </property>
        <property name="placeholderText">
         <string>快捷键</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fastCaptureShortcut">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>连续截图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serialCaptureEdit">
        <property name="text">
         <string>alt+shift+z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="serialCaptureShortcut">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="modeTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="fullScreen">
       <attribute name="title">
        <string>全屏</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="screenArea">
       <attribute name="title">
        <string>区域</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="previewLabel">
        <property name="text">
         <string>预览</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>查看</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>制作</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>345</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionEmoji_Creator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEmoji_Creator">
   <property name="text">
    <string>表情包制造者</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
